src/Caching/Storages/FileStorage.php

```diff
@@ -249,6 +249,7 @@ public function clean(array $conditions): void
 	{
 		$all = !empty($conditions[Cache::ALL]);
 		$collector = empty($conditions);
+		$namespaces = $conditions[Cache::NAMESPACE] ?? false;
```

Comment:
Not sure how you mean it with retyping it to array at top. It wouldn't work even if it were enclosed in brackets.